src/SUFST/src/can_unpack.c

```diff
@@ -10,15 +10,19 @@
 #define QUEUE_RX_THREAD_PRIORITY             10
 #define QUEUE_RX_THREAD_STACK_SIZE           1024
 #define QUEUE_RX_THREAD_PREEMPTION_THRESHOLD 10
+#define STATS_TIMER_TICKS                    100
+#define STATS_TIMER_SECONDS                 STATS_TIMER_TICKS/100
 
 void queue_receive_thread_entry(ULONG input);
+void stats_init(unpack_performance_t* stats);
+void timer_callback(unpack_performance_t* stats);
 
```

Comment:
I missed this before, but these functions should all be `static` as they are internal to this module. I would also call it `stats_timer_callback()` just to be more clear.